### ISSUE TITLE
Background and foreground colors have a sufficient contrast ratio Error! (axe-core Error: Cannot read property 'filter' of undefined)

### ISSUE THREAD

**Issue by sergeykish** [GoogleChrome/lighthouse#11535]
**Describe the bug**
Looks like internal error, issue reported https://github.com/GoogleChrome/lighthouse/issues/11384

**To Reproduce**
1. Go to https://web.dev/measure/
2. Input url http://sergeykish.com/web-api-element-prototype-queryxpathall
3. Click on Run adit
4. Click View Report
5. Scroll down to Accessibility

**Actual result**
* Question mark in circle
* Background and foreground colors have a sufficient contrast ratio Error! (axe-core Error: Cannot read property 'filter' of undefined)

**Expected behavior**
* It should not error
* It should give good result with default browser colors

**Screenshots**
![screenshot](https://user-images.githubusercontent.com/17841/95175828-b6fdc180-07c4-11eb-8cd8-766950cc3e1e.png)

**Comment by patrickhulce** [comment]
Thanks @sergeykish! Looks like you already found the correct issue that tracks this :) 

https://github.com/GoogleChrome/lighthouse/issues/11384 we'll close this as a duplicate of that
